tests/benchdnn/doc/knobs_attr.md

```diff
@@ -8,8 +8,8 @@
     --attr-rounding-mode=ARG:MODE[+...]
     --attr-deterministic=BOOL
     --attr-dropout=PROBABILITY[:SEED[:TAG]]
-    --attr-scales=ARG:POLICY[:SCALE][:DATA_TYPE[:GROUPS]][+...]
-    --attr-zero-points=ARG:POLICY[:ZEROPOINT][:DATA_TYPE[:GROUPS]][+...]
+    --attr-scales=ARG:POLICY[:SCALE[:DATA_TYPE[:GROUPS]]][+...]
+    --attr-zero-points=ARG:POLICY[:ZEROPOINT[:DATA_TYPE[:GROUPS]]][+...]
```

Comment:
The original position was to reflect that `:SCALE` can't be present with `:GROUPS`, but probably, it makes more sense to introduce two lines:
```
--attr-scales=ARG:POLICY[:SCALE[:DATA_TYPE]][+...]
              ARG:POLICY[:DATA_TYPE[:GROUPS]][+...]
```
and same for ZPs.